api/nakadi-event-bus-api.yaml

```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```

Comment:
Then you think I should say in the description that it might be required in special cases?